### PR TITLE
Update Lesson_1_Get_Local_Solana_Env_Running.md

### PR DESCRIPTION
**Pull request by chiarawilden**
A few typos – "it;d" actually got copied and pasted in here while I was searching the doc – will update

### CODE REVIEW

File: Solana_And_Web3/en/Section_2/Lesson_1_Get_Local_Solana_Env_Running.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ solana --version
 
 If that output a version number, you're good to go!
 
-Next thing you'll want to do is run these two commands separately:
+Next thing you'll want to do is run these two commands separately:it;d
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_2/Lesson_1_Get_Local_Solana_Env_Running.md:78]: Looks like a bad copy paste here - `it;d`

**Review by adilanchian — CHANGES_REQUESTED** [review]
Thanks for this!!

Small type change here please :)

**Comment by chiarawilden** [comment]
@adilanchian I think this should fix it – still a little confused by how editing pull requests works.

**Comment by adilanchian** [comment]
You did great @chiarawilden !!

Feel free to DM me in Discord whenever if you have any questions about this stuff, but for now - NICE WORK AND THANK YOU <3

**Comment by adilanchian** [comment]
@all-contributors please add @chiarawilden  for doc

**Pull request merged**